autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/configuration/AutofillRuntimeConfigProvider.kt

```diff
@@ -87,5 +84,9 @@ class RealAutofillRuntimeConfigProvider @Inject constructor(
         }
     }
 
+    private suspend fun determineIfAutofillEnabled(): Boolean {
+        return autofillCapabilityChecker.isAutofillEnabledByConfiguration() && autofillCapabilityChecker.isAutofillEnabledByUser()
```

Comment:
>I don't think you need to check if the config is enabled because that's already done down the line and if is not this will never be called.

Answered here: https://github.com/duckduckgo/Android/pull/2789#discussion_r1094733540



>Also autofillCapabilityChecker.isAutofillEnabledByConfiguration() calls autofillGlobalCapabilityChecker.isAutofillEnabledByConfiguration() so I would just use the latter and make the former non-public

Other parts of the codebase should not have to interrogate `autofillGlobalCapabilityChecker` (or `internalTestChecker`, or the likes...) directly in regards to autofill capabilities; they should use the one API for this, even if that is internally only deferring to something else